daemon/openrazer_daemon/dbus_services/dbus_methods/deathadder_chroma.py

```diff
@@ -377,7 +400,7 @@ def set_scroll_static(self, red, green, blue):
 @endpoint('razer.device.lighting.scroll', 'setScrollBlinking', in_sig='yyy')
 def set_scroll_blinking(self, red, green, blue):
     """
-    Set the device to pulsate
+    Set the device to BLINKING
```

Comment:
Make it lowercase as the rest of the comments

Reply:
Done.